src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -237,7 +237,7 @@ public static class TextCatalog
         /// <param name="inputColumn">The column containing text to remove stop words on.</param>
         /// <param name="outputColumn">The column containing output text. Null means <paramref name="inputColumn"/> is replaced.</param>
         /// <param name="language">Langauge of the input text column <paramref name="inputColumn"/>.</param>
```

Comment:
>/// <param name="language">Langauge of the input text column <paramref name="inputColumn"/>.</param> [](start = 8, length = 100)

did you want to reference the sample from here?

Reply:
Am I doing this right?

---
In reply to: [250825867](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825867) [](ancestors = 250825867)